docs/csharp/whats-new/tutorials/snippets/primary-constructors/BankAccount.cs

```diff
@@ -0,0 +1,115 @@
+﻿namespace primary_constructors;
+
+// <BaseClass>
+public class BankAccount(string accountID, string owner)
```

Comment:
Should tutorial say anything about validation of constructor arguments, which is common practice, I assume. Do primary constructors and argument validation play together somehow?

Reply:
@BillWagner I like your additions in that last commit, thank you!